packages/react-topology/src/components/TopologyView/examples/Topology.md

```diff
@@ -1,155 +1,71 @@
+NodeShape,
+NodeStatus,
+SELECTION_EVENT,
+Visualization,
+VisualizationProvider,
+VisualizationSurface
+} from '@patternfly/react-topology';
+
 import './topology-example.css';
+import Icon1 from '@patternfly/react-icons/dist/esm/icons/regions-icon';
+import Icon2 from '@patternfly/react-icons/dist/esm/icons/folder-open-icon';
 
 ## Examples
-### Default controls
-```js
-import React from 'react';
-import { TopologyView, TopologyControlBar, createTopologyControlButtons, TopologySideBar } from '@patternfly/react-topology';
-import { ProjectToolbar } from './ProjectToolbar';
-import { ViewToolbar } from './ViewToolbar';
-import { ItemDetails } from './ItemDetails';
-
-class DefaultTopologyView extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = { detailsShown: false };
-  }
-
-  render() {
-    const { detailsShown } = this.state;
-    const controlButtons = createTopologyControlButtons();
-    const sideBar = <ItemDetails show={detailsShown} onClose={() => this.setState({ detailsShown: false })} />;
-
-    return (
-      <TopologyView 
-        contextToolbar={<ProjectToolbar />}
-        viewToolbar={<ViewToolbar />}
-        controlBar={<TopologyControlBar controlButtons={controlButtons} />}
-        sideBar={sideBar}
-        sideBarOpen={detailsShown}
-      >
-        <button onClick={() => this.setState({ detailsShown: !detailsShown })}>
-          {detailsShown ? 'Hide Details' : 'Show Details'}
-        </button>
-      </TopologyView>
-    );
-  }
-}
-```
 
-### Selected controls
-```js
-import React from 'react';
-import { TopologyView, TopologyControlBar, createTopologyControlButtons } from '@patternfly/react-topology';
-import { ItemDetails } from './ItemDetails';
-import { ViewToolbar } from './ViewToolbar';
-
-class SelectedTopologyView extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = { detailsShown: false };
-  }
-
-  render() {
-    const { detailsShown } = this.state;
-    const sideBar = <ItemDetails show={detailsShown} onClose={() => this.setState({ detailsShown: false })} />;
-    const controlButtons = createTopologyControlButtons({ fitToScreen: false, legend: false });
-
-    return (
-      <TopologyView
-        viewToolbar={<ViewToolbar />}
-        controlBar={<TopologyControlBar controlButtons={controlButtons} />}
-        sideBar={sideBar}
-        sideBarOpen={detailsShown}
-      >
-        <button onClick={() => this.setState({ detailsShown: !detailsShown })}>
-          {detailsShown ? 'Hide Details' : 'Show Details'}
-        </button>
-      </TopologyView>
-    );
-  }
-}
+### Baseline Topology
+
+1. Declare a controller, which can be initialized via the `useVisualizationController()` method.
+
+1. The `fromModel` method must be called on the controller to create the nodes. `fromModel` will take your data model as a parameter. Your data model should include a `graph` object, on which you will need to set `id` , `type` and `layout`.
+
```

Comment:
Add info about adding the `VisualizationProvider` which is a useful context provider to allow access to the created Controller and is required when using the `VisualizationSurface` component.